lib/bcdice/game_system/SajinsenkiAGuS2E.rb

```diff
@@ -52,6 +56,29 @@ def eval_game_system_specific_command(command)
           roll_tables(command, SECOND_ED_TABLES)
       end
 
+      def roll_ippan(command)
+        result = super
+        return change_fumble(result)
+      end
+
+      def roll_hit_check(command)
+        result = super
+        return change_fumble(result)
+      end
+
+      def change_fumble(result)
+        return nil if result.nil?
+
+        fumble_counts = result.rands.count { |val| val >= 8 }
+        if fumble_counts >= 2
+          result.text.sub!(/(成功|失敗).*$/, 'ファンブル')
```

Comment:
```suggestion
          result.text = result.text.sub(/(成功|失敗).*$/, 'ファンブル')
```

リテラルで記述した文字列がfreezeされるので、Stringクラスでは`!`系の破壊的メソッドを使わない方が得策です。

Reply:
了解しました。修正します。